deprecated.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package hujson
+
+import (
+	"bytes"
+	"io"
+
+	json "github.com/tailscale/hujson/internal/hujson"
+)
+
+// Deprecated: Do not use. This will be deleted in the near future.
```

Comment:
It'd be nice to provide a reason after "Deprecated:", per the standard at https://github.com/golang/go/wiki/Deprecated. Something like "Deprecated: This has been replaced by XXX and will be deleted soon."